application/account-management/Application/AccountRegistrations/StartAccountRegistration.cs

```diff
@@ -20,7 +20,7 @@ public TenantId GetTenantId()
     }
 }
 
-public sealed record StartAccountRegistrationResponse(string AccountRegistrationId, int ValidForSeconds);
+public sealed record StartAccountRegistrationResponse(string AccountRegistrationId, string Email, int ValidForSeconds);
```

Comment:
I don't see this is used in the frontend. It does not belong in the response, as the only thing we need for `CompleteAccountRegistration` is the `AccountRegistrationId`.

Am I missing something?

Reply:
It makes the FE code much cleaner + we can trust that the data in this response